### ISSUE TITLE
App Store - expand availability

### ISSUE THREAD

**Issue by upintheairsheep** [rileytestut/Delta#300]
The App Store version is only available in the United States of America, while I downloaded it myself, please expand the release to other countries that cannot access the alternate App Store functionality that EU countries enjoy, this includes India, China, Australia, Brazil, Nigeria, Indonesia, the Arab World, Russia, Canada, Mexico, UK (due to Brexit), and many others, that cannot download the app without using Windows, macOS, or unofficially Linux, meaning that users without a computer and ChromeOS users that do not live in the US or EU cannot use Delta at all.

**Comment by muuvmuuv** [comment]
I disagree by leaving out EU, I don't want an alt Store on my phone but would love to enjoy the emulator. Apple recently said they allow it, so I don't understand the quote of having it only in AltStore for these countries. Can someone explain further?

**Comment by Ein-Tim** [comment]
@muuvmuuv I think the problem is that even if they distribute Delta via the App Store in the EU they are forced to pay the Core Technology Fee for every download in the EU to apple (first million downloads are free, after that it's 0,50€ per download). This means: You could either put Delta on the App Store in the EU and charge money for it (would probably come out to ca. the 1,50€ we already see for AltStore PAL) or, and that's what @rileytestut and the team probably decided is the preferred solution, you simply don't put it on the EU app store (so that they can keep the app free of charge for everyone in the App Store) and only distribute it via AltStore PAL inside the EU.

But again, the Apple documentation is not written in a very understandable way and I'm also only an outsider making a guess.

**Comment by n3wtype** [comment]
In that case why not charge 1$ per download from App Store for EU based users. I would gladly pay few bucks for AppStore version to avoid any kind of alt store garbage.

**Comment by muuvmuuv** [comment]
Yeah, me as well. That a fee you can pass to the users if they want to stay in AppStore (like me) and don't want third party alt store.

**Comment by Ein-Tim** [comment]
I don't think you can make an app free in some regions and paid in others. You can adjust prices based on different regions, but whether an app is paid or free is a "global" choice, see [Make an iOS free for other countries and not in other countries? - Apple Developer Forums](https://forums.developer.apple.com/forums/thread/22831)

**Comment by muuvmuuv** [comment]
But you can have in-app purchases based on regions, maybe a workaround

**Comment by n3wtype** [comment]
Or just put second paid app globally. Like Delta PRO. With the same features but paid and make it a form of support to developer and project. This way anyone could decide what version they prefer and from which source.

**Comment by upintheairsheep** [comment]
Done.
